.github/workflows/test-lightwalletd.yml

```diff
@@ -0,0 +1,319 @@
+
+  # Test launching lightwalletd with an empty lightwalletd and Zebra state
+  test-lightwalletd-integration:
+    name: Test integration with lightwalletd
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    steps:
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Run tests with empty lightwalletd launch
+        run: |
+          docker pull ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
+          docker run -e ZEBRA_SKIP_IPV6_TESTS -e ZEBRA_TEST_LIGHTWALLETD --name lightwalletd-tests -t ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} cargo test --locked --release --features enable-sentry --test acceptance -- --nocapture lightwalletd_integration
+
+  # Test full sync of lightwalletd with a Zebra tip state
+  lightwalletd-full-sync:
+    name: Test lightwalletd full sync
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    steps:
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Run tests with lightwalletd full sync
+        run: |
+          docker pull ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
+          docker run -e ZEBRA_SKIP_IPV6_TESTS -e ZEBRA_TEST_LIGHTWALLETD -e ZEBRA_CACHED_STATE_DIR -e LIGHTWALLETD_DATA_DIR --name lightwalletd-tests -t ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} cargo test --locked --release --features enable-sentry -- --ignored --nocapture  lightwalletd_full_sync
+        env:
+          ZEBRA_CACHED_STATE_PATH: '$HOME/.cache/zebra'
+          LIGHTWALLETD_DATA_DIR: '$HOME/.cache/lightwalletd'
+
+  # Test update sync of lightwalletd with a lightwalletd and Zebra tip state
+  lightwalletd-update-sync:
+    name: Test lightwalletd update
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    steps:
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Run tests with lightwalletd update sync
+        run: |
+          docker pull ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
+          docker run -e ZEBRA_SKIP_IPV6_TESTS -e ZEBRA_TEST_LIGHTWALLETD -e ZEBRA_CACHED_STATE_DIR -e LIGHTWALLETD_DATA_DIR --name lightwalletd-tests -t ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} cargo test --locked --release --features enable-sentry -- --ignored --nocapture  lightwalletd_update_sync
+        env:
+          ZEBRA_CACHED_STATE_PATH: '$HOME/.cache/zebra'
+          LIGHTWALLETD_DATA_DIR: '$HOME/.cache/lightwalletd'
```

Comment:
These tests can happen in later PRs

Reply:
Fixed by [refactor(ci): use correct test distribution](https://github.com/ZcashFoundation/zebra/pull/4223/commits/70ef046dbd7b428eec6fcc614a76b8ee85c1e24b)

Reply:
It looks like we still have "Test full sync of lightwalletd with a Zebra tip state" in there - can we do it in a separate PR?